src/docfx/lib/moniker/MonikerRangeParser.cs

```diff
@@ -10,18 +10,18 @@ namespace Microsoft.Docs.Build
 {
     internal class MonikerRangeParser
     {
-        private readonly ConcurrentDictionary<string, Lazy<List<string>>> _cache = new ConcurrentDictionary<string, Lazy<List<string>>>();
+        private readonly ConcurrentDictionary<string, Lazy<IEnumerable<string>>> _cache = new ConcurrentDictionary<string, Lazy<IEnumerable<string>>>();
```

Comment:
This can remain List as a private field

Reply:
yes, but it don't have to be, right?

---
In reply to: [236547338](https://github.com/dotnet/docfx/pull/3684#discussion_r236547338) [](ancestors = 236547338)

Reply:
+1, it is enough to keep `IEnumerable<string> Parse(string rangeString)`

Reply:
IReadOnlyList would be better?